src/migrations/20231019233102-cleanup-ongoing-monitoring-topic.js

```diff
@@ -0,0 +1,46 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      const oldTopicId = 127;
+      const newTopicId = 130;
+
+      await queryInterface.sequelize.query(`
```

Comment:
I'm confused as to when we use the "mapTo" column and when we don't

Reply:
The column 'mapTo' should point to the id of the topic that is replacing it. Although I don't think we are doing a lookup anywhere via this column.

![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/e75287c8-4254-43dc-bb90-12d8bb31e0b2)